test/integration-tests/services/l1-ingestion/service.spec.ts

```diff
@@ -0,0 +1,267 @@
+import { expect } from '../../../setup'
+
+/* Imports: External */
+import { ethers } from 'hardhat'
+import { Contract, Signer } from 'ethers'
+import { getContractFactory } from '@eth-optimism/contracts'
+
+/* Imports: Internal */
+import { L1DataTransportService } from '../../../../src/services/main/service'
+import { L1DataTransportClient } from '../../../../src/client/client'
+import { sleep, encodeAppendSequencerBatch } from '@eth-optimism/core-utils'
+
+describe('L1 Data Ingeston Service', () => {
+  let signer: Signer
+  before(async () => {
+    ;[signer] = await ethers.getSigners()
```

Comment:
```suggestion
    ([signer] = await ethers.getSigners())
```
Is there a preferred style between these approaches?

Reply:
We're using the semicolon style in our other repos right now but am open to change this in the future. Will make a note of this.